src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -11,7 +11,7 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "c-clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "Employees' contact list has been cleared!";
```

Comment:
Instead of using "Employees' contact list", I'm using "Employee directory", as we now use some tags to help on manpower management, so it also stores other info (eg. available working days, etc) on top of contact information. So you may want to change to "Employee directory" instead? Thanks 😃

Reply:
ok! now